graviti/dataframe/frame.py

```diff
@@ -168,7 +175,39 @@ def from_lazy_lists(cls: Type[_T], lazy_lists: "LazyLists") -> _T:
             The loaded :class:`~graviti.dataframe.DataFrame` object.
 
         """
-        return cls(lazy_lists)
+        # Not support loading tensorbay Dataset.
+        return cls(lazy_lists)  # type: ignore[arg-type]
+
+    @classmethod
+    def from_pyarrow(cls: "Type[DataFrame]", array: pa.StructArray, schema: record) -> "DataFrame":
```

Comment:
we should use TypeVar for cls and return type,
if DataFrame has a subclass and we use "DataFrame" as the Type, the type will mismatch in subclass